environment.yml

```diff
@@ -0,0 +1,25 @@
+name: nbgrader-dev
```

Comment:
Either way, If we take a ``pip`` approach or ``conda`` approach I would really like to have this file in the repo as it would make it easier to setup a fresh conda dev env for nbgrader.